ts/features/fims/store/reducers/index.ts

```diff
@@ -8,16 +7,17 @@ import {
   fimsGetRedirectUrlAndOpenIABAction
 } from "../actions";
 import { ConsentData } from "../../types";
+
+type FimsFlowStateTags = "consents" | "in-app-browser";
+
 export type FimsState = {
-  ctaUrl?: string;
+  currentFlowState: FimsFlowStateTags;
   consentsData: pot.Pot<ConsentData, Error>;
```

Comment:
If the `error` is always a string message, what about using a string directly in the pot's error state instead of an Error?

Reply:
sure, why not? the only downside I could see is that as of right now seeing "string" in the typing could be not so understandable, but in the future there will be an error map so no biggie 👍